### PR TITLE
Set default join method in combobox to left

### PR DESCRIPTION
**Pull request by tmelliott**
I've just reordered some things - set the default value within the "gcombobox" widget itself, and then fetch the value of that as the global "join_method" variable. 

@yiwen16  if you have time could you pull this "hotfix" branch and make sure it works with your various test files. If you're too busy just let me know and I'll do it when I get a chance. Thanks.

### CODE REVIEW

**Comment by yiwen16** [comment]
Hi,
I have run some tests on the function and changed the code slightly and
have pushed everything to the hotfix branch, should I create a pull request
now?

**Comment by tmelliott** [comment]
@yiwen16 PR is already created. Thanks.

File: R/iNZChangeDataWin.R [hunk 1]
```diff
@@ -1022,12 +1022,9 @@ iNZjoinDataWin <- setRefClass(
         string1 <- glabel("Preview of the original dataset")
         originview <- gtable(data.frame(head(GUI$getActiveData(), 10)))
         string2 <- glabel("Select join methods")
-        var1 <- gcombobox(
-	    items = c("Inner Join", "Left Join", "Full Join", "Semi Join", "Anti Join"),
-	    selected = 2
-        )
-	join_method <<- svalue(var1)
-
+        var1 <- gcombobox(items = c("Inner Join", "Left Join", "Full Join", "Semi Join", "Anti Join"), selected = 2)
```
Comment (tmelliott) [inline R/iNZChangeDataWin.R]: This should be OK for now. Should probably create a mapping table to make it easier to change this in future.
```r
jointypes <- list("Inner Join" = "inner_join", "Left Join" = "left_join", ...)
var1 <- gcombobox(items = names(jointypes), selected = 2) # this is all you now need to change to adjust default
join_method <- jointypes[[svalue(var1)]]
```

Comment (tmelliott) [reply R/iNZChangeDataWin.R]: Your switch later down could then use the jointypes also.

**Pull request merged**